docs/t-sql/functions/char-transact-sql.md

```diff
@@ -176,8 +178,11 @@ single_byte_representing_complete_character single_byte_representing_complete_ch
 ｼ                                           ｼ                                         
 ```
 
-### F. Using CHAR to return multibyte characters  
-This example uses the an integer and hex values in the valid range for ASCII. However, the CHAR function returns NULL because the parameter represents only the first byte of a multibyte character.
+### F. Using CHAR to return multibyte characters
+This example uses integer and hex values in the valid range for Extended ASCII.
+However, the `CHAR` function returns `NULL` because the parameter represents only the first byte of a multibyte character.
+A **char(2)** double-byte character cannot be composed from or divided into parts without conversion:
+those parts are not generally valid **char(1)** values.

```

Comment:
One final suggestion for clarity. Would you change:

"A **char(2)** double-byte character cannot be composed from or divided into parts without conversion: those parts are not generally valid **char(1)** values. "

To:

"A **CHAR(2)** double-byte character cannot be partially represented nor divided without some conversion operation. The individual bytes of a double-byte character don't generally represent valid **CHAR(1)** values."

Hopefully I've captured your intent properly.

Reply:
I hope its okay if I snuck in a "neither" to go with your "nor". I've also kept the lowercase on the datatype since that seems to be the dominant convention. Should I change that?